localstack/utils/cloudwatch/cloudwatch_util.py

```diff
@@ -27,14 +27,18 @@ def publish_lambda_metric(metric, value, kwargs):
     if not config.service_port('cloudwatch'):
         return
     cw_client = aws_stack.connect_to_service('cloudwatch')
-    cw_client.put_metric_data(Namespace='AWS/Lambda',
-        MetricData=[{
-            'MetricName': metric,
-            'Dimensions': dimension_lambda(kwargs),
-            'Timestamp': datetime.now(),
-            'Value': value
-        }]
-    )
+    try:
+        cw_client.put_metric_data(Namespace='AWS/Lambda',
+            MetricData=[{
+                'MetricName': metric,
+                'Dimensions': dimension_lambda(kwargs),
+                'Timestamp': datetime.now(),
+                'Value': value
+            }]
+        )
+    except:
+        print("INFO: cloudwatch has not started, it seems.")
```

Comment:
Can we please use `LOG.info` here - we generally try to avoid using `print` in most places.

Also, let's please use a more informative log message, something like:
```
...
except Exception as e:
    LOG.info('Unable to put metric data for metric "%s" to CloudWatch: %e' % (metric, e))
```

Reply:
okay, sure.